File: problems/java/MeetingScheduler/src/enums/NotificationType.java
package enums;

public enum NotificationType {
    INVITE("Invite", "Meeting Invitation"),
    CANCEL("Cancel", "Meeting Cancelled");

    private final String label;
    private final String subjectPrefix;

    NotificationType(String label, String subjectPrefix) {
        this.label = label;
        this.subjectPrefix = subjectPrefix;
    }

    public String getLabel() {
        return label;
    }

    public String getSubjectPrefix() {
        return subjectPrefix;
    }
}

File: problems/python/MeetingScheduler/src/enums/notification_type.py
from enum import Enum


class NotificationType(Enum):
    INVITE = ("Invite", "Meeting Invitation")
    CANCEL = ("Cancel", "Meeting Cancelled")

    def __init__(self, label: str, subject_prefix: str) -> None:
        self.label = label
        self.subject_prefix = subject_prefix
